test-cases-internal/src/main/java/com/palantir/conjure/verification/GenerateClientServices.java

```diff
@@ -55,14 +59,22 @@ private static void writeServiceDefinition(
     private static Map<String, Object> generateAutoDeserializeService(List<BodyTests> bodyTests) {
         ImmutableMap.Builder<String, Object> endpoints = ImmutableMap.builder();
 
-        bodyTests.stream().map(BodyTests::getType).map(TestCasesUtils::parseConjureType).forEach(conjureType -> {
-            String endpointName = TestCasesUtils.typeToEndpointName("get", conjureType);
-            String typeName = conjureType.visit(new ResolveLocalReferencesConjureTypeVisitor());
-            endpoints.put(endpointName, ImmutableMap.of(
-                    "http", "POST /" + endpointName,
-                    "returns", typeName,
-                    "args", ImmutableMap.of("body", typeName)));
-        });
+        bodyTests.stream()
+                .map(BodyTests::getType)
+                .map(TestCasesUtils::parseConjureType)
+                .forEach(conjureType -> {
+                    String endpointName = TestCasesUtils.typeToEndpointName("get", conjureType);
+                    String typeName = conjureType.visit(new ResolveLocalReferencesConjureTypeVisitor());
+                    endpoints.put(
+                            endpointName,
+                            ImmutableMap.of(
+                                    "http",
+                                    "POST /" + endpointName,
+                                    "returns",
+                                    typeName,
+                                    "args",
+                                    ImmutableMap.of("body", typeName)));
```

Comment:
as discussed before here this is not great but I think we can move forward with the PR for now.
We are tracking the issue at https://github.com/palantir/palantir-java-format/issues/204